contributing/content-style-guide.md

```diff
@@ -18,11 +18,51 @@ This guide includes standards we want to emphasize, likely because we've made de
 * [Images](#Images)
 * [Talk to us](#Talk-to-us)
 
+## Folder Structure and TOC
+
+The folder structure for the [docs.getdbt.com](https://github.com/dbt-labs/docs.getdbt.com) repository is organized into several high-level categories under the main `website` folder: blog, cypress, docs, functions, plugins, snippets, src, static.
+
+The table of contents (TOC) is managed in the `sidebar.js` file. You only need to edit the `sidebar.js` file when you are adding a new page or deleting an existing page. The `sidebar.js` file is the one that causes most of the merge conflicts because many technical writers are working on content daily. You will need to accept the changes from other contributors if you are committing a PR.
+
+Don't worry if you're not sure where in the TOC a new topic belongs. Do your best and when you submit your PR, the dbt Labs Documentation team will edit it and help to find the right placement.
+
+The right-hand TOC is created automatically when you add headings to a page.
+
+## Filenaming
+
+If you are adding a new file in the [docs.getdbt.com](https://github.com/dbt-labs/docs.getdbt.com) repository, it must be named following our naming conventions. MORE TO ADD
```

Comment:
yes def! we can try to decide on this async but that's another one of my questions in the description up top!  maybe i can start a thread in slack to get it started?

Reply:
Ideas for content once polling is finished:

When you create or rename a file or directory and it has more than one word in its name, use dashes (-) instead of spaces or underscores. 

**Option 1:**
The file name should always start with a verb and the feature type, like ide, scheduler, or python. Depending on the content type, it typically also includes a secondary descriptor. Verbs are used when you are creating a procedural document. 

Due to the fact that we author content using Markdown, you must add the `.md` file extension to the file name. For example, `ide-features.md`

If you are adding a new file to an existing feature category, follow the current naming convention.

| File type | Example |
| :-------: | :-------: |
| Conceptual file | `set-up-dbt-cloud.md` |
| Procedural file | `set-up-cloud-environment.md` |
| Reference file | `exclude-flag.md` |

**Option 2:**
The file name should always start with the feature type, like ide, scheduler, or python. Depending on the content type, it typically also includes a secondary descriptor or a verb. Verbs are used when you are creating a procedural document. 

Due to the fact that we author content using Markdown, you must add the `.md` file extension to the file name. For example, `ide-features.md`

If you are adding a new file to an existing feature category, follow the current naming convention.

| File type | Example |
| :-------: | :-------: |
| Conceptual file | `dbt-cloud-set-up.md` |
| Procedural file | `cloud-environment-set-up.md` |
| Reference file | `exclude-flag.md` |